core/treepath/treepath.go

```diff
@@ -177,3 +177,16 @@ func (tp Treepath) Apply(n *game.Node) *game.Node {
 	}
 	return curNode
 }
+
+// Returns the treepath as a string. 
+// examples: 
+// 	[]				becomes "[]"
+// 	[1]				becomes	"[1]"
+// 	[1,2,0,2,2,2] 	becomes "[1,2,0,2,2,2]"
+func (tp Treepath) String() string {
+	var strArr []string
+	for _, i := range tp{
```

Comment:
formatting is a bit off. gofmt will automatically fix this, but it needs to be configured. What editor are you using?

Reply:
VScode

Reply:
I would check out: https://code.visualstudio.com/docs/languages/go. Especially enable the autoformatting / linting / vet to start with =)